rocket-cli/src/gs/mod.rs

```diff
@@ -170,7 +170,7 @@ pub fn tui_task(
                             .child(
                                 LinearLayout::vertical()
                                     .child(
-                                        EditView::new().content("4000").with_name("target_apogee"),
+                                        EditView::new().content("").with_name("target_apogee (meters)"),
```

Comment:
this is the wrong place. the visible label for the text field is above in line `TextView::new("Target Altitude: ")`